installer/install.iss

```diff
@@ -2788,16 +2810,28 @@ begin
         GitSystemConfigSet('pull.ff','only')
     end;
 
+    {
+        Configure credential helper
+    }
+
+    if RdbGitCredentialManager[GCM_None].checked then begin
+        GitSystemConfigSet('credential.helper',#0);
```

Comment:
I'm not sure if we want the option to be "None" which forcibly removes any configured helper, or change it to "Unset" which does nothing?

Reply:
Since we're talking about the system config, I think `None` is appropriate (and technically, the same as `Unset`): it's not like we leave previously-configured settings in there, otherwise you would not be able to turn off GCM during a reinstallation.